### PR TITLE
Fix imports in std.algorithm

### PR DESCRIPTION
**Pull request by 9il**
remove dependency from `std.range`

### CODE REVIEW

File: std/algorithm/mutation.d [hunk 1]
```diff
@@ -75,9 +75,6 @@ import std.traits : isArray, isBlitAssignable, isNarrowString, Unqual;
 // FIXME
 import std.typecons; // : tuple, Tuple;
 
-// FIXME: somehow deleting this breaks the bringToFront() unittests.
-import std.range;
-
```
Comment (greenify) [inline std/algorithm/mutation.d]: Nice! I just wanted to send a PR for this too ;-)

**Comment by JackStouffer** [comment]
I don't understand the rationale for these changes. What do we get in return for making the code a lot less readable?

**Comment by 9il** [comment]
We remove the import of `std.range`

File: std/algorithm/mutation.d [hunk 2]
```diff
@@ -102,68 +99,68 @@ reachable from $(D front) and $(D front) is not reachable from $(D
...
+    b = a forward range
 
 Returns:
     The number of elements brought to the front, i.e., the length of $(D back).
 
 See_Also:
     $(WEB sgi.com/tech/stl/_rotate.html, STL's rotate)
 */
-size_t bringToFront(Range1, Range2)(Range1 front, Range2 back)
+size_t bringToFront(Range1, Range2)(Range1 a, Range2 b)
```
Comment (DmitryOlshansky) [inline std/algorithm/mutation.d]: front and back did make sense unlike a and b

**Comment by DmitryOlshansky** [comment]
I suggest to split off deprecations as a separate pull

**Comment by 9il** [comment]
done

**Comment by DmitryOlshansky** [comment]
LGTM

**Comment by JackStouffer** [comment]
> We remove the import of std.range

Why?

**Comment by 9il** [comment]
> Why?

To reduce import dependencies between phobos modules.

**Comment by DmitryOlshansky** [comment]
Auto-merge toggled on

**Pull request merged**